docs/git-comparison.md

```diff
@@ -127,13 +127,24 @@ parent.
     <tr>
       <td>Show diff of the current change</td>
       <td><code>jj diff</code></td>
-      <td><code>git diff HEAD</code></td>
+      <td><code>git diff</code></td>
     </tr>
     <tr>
       <td>Show diff of another change</td>
       <td><code>jj diff -r &lt;revision&gt;</code></td>
       <td><code>git diff &lt;revision&gt;^ &lt;revision&gt;</code></td>
     </tr>
+    <tr>
+      <td>Show diff from one change to the current change</td>
+      <td><code>jj diff --from &lt;revision&gt;</code></td>
+      <td><code>git diff &lt;revision&gt;</code></td>
+    </tr>
+    <tr>
+      <td>Show diff from change A to change B</td>
+      <td><code>jj diff --from A --to B</code></td>
+      <td><code>git diff A B </code></td>
+    </tr>
+    <tr>
     <tr>
       <td>Show description and diff of a change</td>
       <td><code>jj show &lt;revision&gt;</code></td>
```

Comment:
This is unrelated to your change, but I think the git version of "Remove file in the current change" should be `git rm file`, right?

Reply:
Yes. Will submit another PR for this.

Reply:
Sounds good!

Reply:
Done in #1848.